### ISSUE TITLE
Wrong illustration in README.md

### ISSUE THREAD

**Issue by yuantailing** [pytorch/pytorch#98600]
### 📚 The doc issue

The last frame of the illustration is wrong:

![wrong illustration](https://github.com/pytorch/pytorch/raw/master/docs/source/_static/img/dynamic_graph.gif)

### Suggest a potential alternative/fix

The correct illustration should look like this:

![correct illustration](https://user-images.githubusercontent.com/8487114/230635413-925f0858-7166-4557-bb46-484be4b8f1f2.png)

**Comment by FarahSabbagh** [comment]
but unfortunately i didn't understand exactly why this is wrong? it would be nice if you could explain that :)

**Comment by yuantailing** [comment]
@FarahSabbagh The `loss` should be `next_h.sum()`, but it is directly pointing to `next_h` in the animation.

**Comment by drisspg** [comment]
I don't think this is right either, there should be a chain going tanh -> next_h -> sum()

**Comment by yuantailing** [comment]
@drisspg Similar to how `Add` is linked to `MM` rather than `h2h` and `i2h`, I think `Sum` may be linked to `Tanh`.
